tests/testBasisLists.py

```diff
@@ -182,42 +184,42 @@ def testCentralRegularization(self):
         try:
             self.policyDF.set("centralRegularizationStencil", 1)
             ipDiffim.makeRegularizationMatrix(self.policyDF)
-        except Exception, e:  # stencil of 1 not allowed
+        except Exception as e:  # stencil of 1 not allowed
```

Comment:
I think this would be cleaner as:
```python
with self.assertRaises(Exception):
    self.policyDF.set("centralRegularizationStencil", 1)
    ipDiffim.makeRegularizationMatrix(self.policyDF)
```
wouldn't it?